app/components/cards/PostSummary.jsx

```diff
@@ -149,7 +149,7 @@ class PostSummary extends React.Component {
         let thumb = null;
         if(pictures && p.image_link) {
           const prox = $STM_Config.img_proxy_prefix
-          const size = (thumbSize == 'mobile') ? '640x480' : '128x256'
+          const size = (thumbSize == 'mobile') ? '640x480' : '256x384';
```

Comment:
The res size here shouldn't be altering the aspect/cropping of the thumbnail(check out the examples).  Theoretically if we can use the smaller resolution, benefit with a smaller size img to serve, and eliminate the bad quality thumbnails on the higher displays.